addok/ds.py

```diff
@@ -56,7 +56,9 @@ def store_documents(docs):
     for doc in docs:
         if not doc:
             continue
-        key = keys.document_key(doc['id'])
+        if '_id' not in doc:
```

Comment:
@yohanboniface and the documentation https://github.com/addok/addok/pull/307/files#diff-648a1879a1bd05c8f82dcd828b2a9c84R18